src/tbbmalloc/CMakeLists.txt

```diff
@@ -73,19 +74,26 @@ set_target_properties(tbbmalloc PROPERTIES
 
 set(CMAKE_CXX_IMPLICIT_LINK_LIBRARIES "")
 
+set_property(TARGET tbbmalloc APPEND PROPERTY LINK_FLAGS_RELWITHDEBINFO )
```

Comment:
Is this line still relevant?

Reply:
No, I forgot to remove this code, thanks for the comment!